src/mon/Monitor.cc

```diff
@@ -5887,7 +5887,7 @@ void Monitor::new_tick()
 void Monitor::tick()
 {
   // ok go.
-  dout(11) << "tick" << dendl;
+  dout(10) << "tick begin" << dendl;
```

Comment:
I'd prefer we use the `ceph::coarse_mono_clock` for this and any new purpose. (No need to update the `now` variable below, just make a new second one.)